src/node/mod.rs

```diff
@@ -6,26 +6,40 @@
 // KIND, either express or implied. Please review the Licences for the specific language governing
 // permissions and limitations relating to use of the SAFE Network Software.
 
+mod command;
 pub mod event_stream;
 mod executor;
 mod stage;
+#[cfg(test)]
+mod tests;
 
 pub use self::event_stream::EventStream;
-use self::{executor::Executor, stage::Stage};
+#[cfg(test)]
+use self::stage::State;
+use self::{
+    command::Command,
+    executor::Executor,
+    stage::{Approved, Bootstrapping, Comm, NodeInfo, Stage},
+};
+#[cfg(all(test, feature = "mock"))]
+use crate::section::SectionKeyShare;
 use crate::{
-    crypto::{name, Keypair, PublicKey},
+    crypto::{Keypair, PublicKey},
     error::{Error, Result},
+    event::{Connected, Event},
     location::{DstLocation, SrcLocation},
     network_params::NetworkParams,
     peer::Peer,
-    rng::MainRng,
+    rng,
     section::{EldersInfo, SectionProofChain},
     TransportConfig,
 };
 use bytes::Bytes;
 use itertools::Itertools;
+#[cfg(feature = "mock")]
```

Comment:
is the plan to bring back the `mock` feature?
However I don't see it be added in the Cargo.toml ?

Reply:
No, this was left here by accident. Will remove.